webapp/sources/rudder/rudder-web/src/main/resources/configuration.properties.sample

```diff
@@ -410,6 +410,21 @@ rudder.dir.gitRoot=/var/rudder/configuration-repository
 #
 rudder.config.repo.new.file.group.owner=rudder
 
+#
+# Periodically do a Git GC to avoid accumulation of garbage.
+# The schedule is defined based on a cron-like format with the usual fields plus a second one:
```

Comment:
"a second one" is ambiguous here, maybe  "with the usual fields, preceded by a field for seconds".